book/website/reproducible-research/testing/testing-exceptions.md

```diff
@@ -12,7 +12,7 @@ Testing this kind of code can be very difficult because if it is run multiple ti
 Random number seeds are a little difficult to explain so here's an example.
 Here's a little Python script that prints three random numbers.
 
-    ```
+    ```Python
```

Comment:
This still doesn't render correctly, could you please check?

Reply:
Fixed